eng/build.ps1

```diff
@@ -353,7 +353,7 @@ if ($env:PATH -notlike "*${env:JAVA_HOME}*") {
 }
 
 if (-not $foundJdk -and $RunBuild -and ($All -or $BuildJava) -and -not $NoBuildJava) {
-    Write-Error "Could not find the JDK. Either run $PSScriptRoot\eng\scripts\InstallJdk.ps1 to install for this repo, or install the JDK globally on your machine (see $PSScriptRoot\docs\BuildFromSource.md for details)."
+    Write-Error "Could not find the JDK. Either run $PSScriptRoot\scripts\InstallJdk.ps1 to install for this repo, or install the JDK globally on your machine (see $PSScriptRoot\..\docs\BuildFromSource.md for details)."
```

Comment:
Need to fix https://github.com/dotnet/aspnetcore/blob/d82046a8648a65fd1f99fda81ed5376a175902eb/eng/build.ps1#L308 too. Could use `$ToolsDir` there if moved below the import of tools.ps1